feat: integrations page

n05la3 opened the pull request:
Closes #20 
Closes #81

n05la3 committed:
refactor(index page): change astronaut left and right hands

n05la3 committed:
fix(search form): ensure search remains visible when a search result item is clicked

n05la3 committed:
feat(IntegrationPage): add integrations page

n05la3 committed:
refactor(sidebar): update color of plug-not-connected svg icon

n05la3 committed:
feat(IntegrationPage): add page titles

IlCallo commented on docs/src/components/landing-page/SearchQuasarForm.vue:
```diff
@@ -118,6 +118,10 @@ export default {
       emit('focus-by-keyboard', scope.focusByKeyboard.value)
     })
 
+    watch(() => scope.searchResults.value, () => {
```
I think `watchEffect` would work in this case, to cleanup the code a bit

```ts
watchEffect(() => emit('search-result-change', scope.searchResults.value))
```

IlCallo commented on docs/src/router/routes.js:
```diff
@@ -8,6 +8,11 @@ const docsPages = [
+  {
+    path: 'integrations',
+    name: 'integrations',
+    component: () => import('pages/landing-page/QuasarIntegrations.vue')
```
This should be together with all other docs pages (I mean the file path), it's not like components one or the landing, which have it's own branding and management

IlCallo commented on docs/src/pages/landing-page/QuasarIntegrations.vue:
```diff
@@ -0,0 +1,175 @@
+<template>
+  <q-page class="column justify-center items-center bg-lp-light-grey q-px-xl font-monserrat">
```
All docs pages should have content bg as `bg-lp-light-grey`, as noted into https://github.com/dreamonkey/quasar/issues/81

IlCallo replied:
We also need to raise the distance between content and the header for all docs pages, as noted into https://github.com/dreamonkey/quasar/issues/66

n05la3 replied:
Fixed #81, the fix for #66 in a subsequent PR.

IlCallo committed:
fix(integrations): small fixes

IlCallo committed:
fix(meta): remove redundant title prefix

n05la3 committed:
fix(landing-page): random fixes
- prefer watchEffect to watch for emitting search form changes
- fixes #81
- change route path file
